src/zeek.bif

```diff
@@ -5328,3 +5328,35 @@ function table_pattern_matcher_stats%(tbl: any%) : MatcherStats
 
 	return std::move(result);
 	%}
+
+## Determine the path used by a non-relative @load directive.
+##
+## This function is package aware: Passing *package* will yield the
+## path to *package.zeek*, *package/__load__.zeek* or an empty string
+## if neither can be found. Note that passing a relative path is an
```

Comment:
Should this be "passing a relative or absolute path"? The if statement below is checking for a path starting with `/` and has a similar error message.